hippo-001.md

```diff
@@ -0,0 +1,95 @@
+# HIPPO-001
+
+```
+HIPPO: 1
+Title: HIPPO Purpose and Guidelines
+Author: John C. Vernaleo  <jcv@hemi.xyz>
+Comments-Summary: No comments yet.
+Comments-URI: NA
+Status: New
+Type: Process
+Created: 2025-10-14
+```
+
+## What is a HIPPO
+
+HIPPO is a Hemi Improvement ProPOsal.  These are design documents to
```

Comment:
I have a hard time reading this for some reason. Maybe using bold to indicate which letters form HIPPO would be easier to read? 🤔 
```suggestion
HIPPO is a **H**emi **I**mprovement **P**ro**po**sal.  These are design documents to
```
> HIPPO is a **H**emi **I**mprovement **P**ro**po**sal.